apple/LiveMarkdownModule.mm

```diff
@@ -16,10 +15,7 @@
 // I think this is the easiest way to access the UIManager, which we need to
 // actually register the hook
```

Comment:
Let's remove this comment as well

Reply:
[af0d6da](https://github.com/Expensify/react-native-live-markdown/pull/566/commits/af0d6daa5ecc42e8084bcfee239e9fca27edd87a)